doc/model/train-fitting-property.md

```diff
@@ -0,0 +1,195 @@
+# Fit other properties {{ pytorch_icon }} {{ jax_icon }} {{ dpmodel_icon }}
+
+:::{note}
+**Supported backends**: PyTorch {{ pytorch_icon }}, JAX {{ jax_icon }}, DP {{ dpmodel_icon }}
+:::
+
+Here we present an API to DeepProperty model, which can be used to fit other properties like band gap, bulk modulus, critical temperature, etc.
+
+In this example, we will show you how to train a model to fit properties of `humo`, `lumo` and `band gap`. A complete training input script of the examples can be found in
+
+```bash
+$deepmd_source_dir/examples/property/train
+```
+
+The training and validation data are also provided our examples. But note that **the data provided along with the examples are of limited amount, and should not be used to train a production model.**
+
+Similar to the `input.json` used in `ener` mode, training JSON is also divided into {ref}`model <model>`, {ref}`learning_rate <learning_rate>`, {ref}`loss <loss>` and {ref}`training <training>`. Most keywords remain the same as `ener` mode, and their meaning can be found [here](train-se-atten.md). To fit the `property`, one needs to modify {ref}`model[standard]/fitting_net <model[standard]/fitting_net>` and {ref}`loss <loss>`.
+
+## The fitting Network
+
+The {ref}`fitting_net <model[standard]/fitting_net>` section tells DP which fitting net to use.
+
+The JSON of `property` type should be provided like
+
+```json
+	"fitting_net" : {
+		"type": "property",
+        "intensive": true,
+        "property_name": "band_prop",
+        "task_dim": 3,
+		"neuron": [240,240,240],
+		"resnet_dt": true,
+		"fparam": 0,
```

Comment:
Do you mean `numb_fparam`? `fparam` seems to be an invalid argument.

Reply:
I have deleted this parameter. Bacause `fparam` is not related to this PR.